dataprofiler/data_readers/graph_differentiator.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import random
+import re
+from collections import Counter
+from io import BytesIO
+from readline import add_history
+from dataprofiler.data_readers.csv_data import CSVData
+from six import StringIO
+import os
+
+#from . import data_utils
+#from .avro_data import AVROData
+#from .json_data import JSONData
+#from .parquet_data import ParquetData
+from .base_data import BaseData
+#from .structured_mixins import SpreadSheetDataMixin
+
+class GraphDifferentiator():
```

Comment:
I see the rational for calling it a `Differentiator` but my suggestion is to keep consistent naming with what is in the `data_readers` and call it something like `CSVData` or something to that effect.... and the file named `graph_data.py`